std-bits/table/src/main/java/org/enso/table/expressions/ExpressionVisitorImpl.java

```diff
@@ -0,0 +1,342 @@
+    final Value type = module.invokeMember("get_type", typeName);
+    this.getMethod = name -> module.invokeMember("get_method", type, name);
+
+    this.variableArgumentFunctions = new HashSet<>(Arrays.asList(variableArgumentFunctions));
+  }
+
+  private Value wrapAsColumn(Value value) {
+    if (value.isNull()) {
+      return makeConstantColumn.apply(value);
+    }
+
+    var metaObject = value.getMetaObject();
+    return metaObject != null && metaObject.asHostObject() instanceof Class<?>
+        ? makeConstantColumn.apply(value)
+        : value;
+  }
+
+  private Value executeMethod(String name, Value... args) {
+    Value method = getMethod.apply(name);
+    if (!method.canExecute()) {
+      throw new UnsupportedOperationException(name);
+    }
+
+    Object[] objects;
+    if (this.variableArgumentFunctions.contains(name)) {
+      objects = new Object[2];
+      objects[0] = args[0];
+      objects[1] = Arrays.copyOfRange(args, 1, args.length, Object[].class);
+    } else {
+      objects = Arrays.copyOf(args, args.length, Object[].class);
+    }
+    objects[0] = wrapAsColumn(args[0]);
+
+    try {
+      var result = method.execute(objects);
+      if (result.canExecute()) {
+        throw new IllegalArgumentException("Insufficient arguments for method " + name + ".");
+      }
+      return result;
+    } catch (PolyglotException e) {
+      if (e.getMessage().startsWith("Type error: expected a function")) {
+        throw new IllegalArgumentException("Too many arguments for method " + name + ".");
+      }
+      throw e;
+    }
+  }
+
+  @Override
+  public Value visitProg(ExpressionParser.ProgContext ctx) {
+    Value base = visit(ctx.expr());
+    return wrapAsColumn(base);
+  }
+
+  @Override
+  public Value visitColumn(ExpressionParser.ColumnContext ctx) {
+    var text = ctx.getText();
+    return getColumn.apply(text.substring(1, text.length() - 1).replace("]]", "]"));
+  }
+
+  @Override
+  public Value visitPower(ExpressionParser.PowerContext ctx) {
+    return executeMethod("^", visit(ctx.expr(0)), visit(ctx.expr(1)));
+  }
+
+  @Override
+  public Value visitMultDivMod(ExpressionParser.MultDivModContext ctx) {
+    return executeMethod(ctx.op.getText(), visit(ctx.expr(0)), visit(ctx.expr(1)));
+  }
+
+  @Override
+  public Value visitCompare(ExpressionParser.CompareContext ctx) {
+    var op = ctx.op.getText();
+    if (op.equals("=")) {
+      op = "==";
+    }
+    if (op.equals("<>")) {
+      op = "!=";
+    }
+
+    return executeMethod(op, visit(ctx.expr(0)), visit(ctx.expr(1)));
+  }
+
+  @Override
+  public Value visitLike(ExpressionParser.LikeContext ctx) {
+    var condition = executeMethod("like", visit(ctx.expr(0)), visit(ctx.expr(1)));
+    return ctx.NOT_LIKE() != null ? executeMethod("not", condition) : condition;
+  }
+
+  @Override
+  public Value visitIsNull(ExpressionParser.IsNullContext ctx) {
+    var op = ctx.IS_NULL() != null | ctx.IS_NOT_NULL() != null ? "is_missing" : "is_empty";
```

Comment:
```suggestion
    var op = ctx.IS_NULL() != null || ctx.IS_NOT_NULL() != null ? "is_missing" : "is_empty";
```

Is the bitwise or delibarate here?